ui/src/join_game/overlay_components.tsx

```diff
@@ -52,6 +57,42 @@ export const ActiveHeisterKeyboardComponent = ({
   );
 };
 
+type TapButtonComponentProps = {
+  name_prefix: string;
+};
+export const TapButtonComponent = ({
+  name_prefix,
+}: TapButtonComponentProps) => {
+  const dispatch = useDispatch();
+  const player_name = useSelector(playerNameSelector);
+
+  const onClick = (_event) => {
+    let recipient = name_prefix.split("'").slice(0, -1).join("'");
+    let msg = `${player_name}: tap ${recipient}`;
+    dispatch(sendChat(msg)); // todo
+  };
+
+  const is_self = name_prefix === "Your";
+  const pointer_events = is_self ? "none" : "auto";
+  const text = is_self ? "-" : "Tap";
+  const class_name = is_self ? "invisible" : undefined;
```

Comment:
Why do we bother rendering the self button at all? Isn't it fine for it to be hidden instead of invisible if we no one would ever click it?